src/main/kotlin/pl/newbies/plugins/Metrics.kt

```diff
@@ -0,0 +1,25 @@
+package pl.newbies.plugins
+
+import io.ktor.server.application.Application
+import io.ktor.server.application.call
+import io.ktor.server.application.install
+import io.ktor.server.metrics.micrometer.MicrometerMetrics
+import io.ktor.server.response.respond
+import io.ktor.server.routing.get
+import io.ktor.server.routing.routing
+import io.micrometer.prometheus.PrometheusConfig
+import io.micrometer.prometheus.PrometheusMeterRegistry
+
+fun Application.configureMetrics() {
+    val appMicrometerRegistry = PrometheusMeterRegistry(PrometheusConfig.DEFAULT)
+
+    install(MicrometerMetrics) {
+        registry = appMicrometerRegistry
+    }
+
+    routing {
+        get("/metrics") {
+            call.respond(appMicrometerRegistry.scrape())
+        }
+    }
+}
```

Comment:
is `/metrics` open to the public?

Reply:
Yup, currently it is and i'll block it in the future